### PR TITLE
animation changes

### PR DESCRIPTION
**Pull request by stri4759**
the first animations for the website for the typing animation on the log in page

### CODE REVIEW

File: incskill_site/templates/login.html [hunk 1]
```diff
@@ -10,12 +10,20 @@
...
     <img src="{% static 'media/incskill.png'%}" alt="Incskill Logo" id="img"></h1>
-    <div>
-      <h2 id="mission">
-        Unlocking high-growth career opportunities irrespective of past experience or needs
+    
+      
+        
+        
+         
+
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:18]: why are there like 82945678902346809 newlines LOL

File: incskill_site/templates/login.html [hunk 2]
```diff
@@ -90,15 +127,16 @@ <h1 id="title">Login</h1>
...
 
   #login-details{
-    margin-left: -60%;
+   margin-left: 60%;
     width: 175%;
     height:50%;
-    margin-top:-20%;
-    position: relative;
+    margin-top: -60%;
+    position: static;
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:139]: you can remove this because static is the default

File: incskill_site/templates/login.html [hunk 3]
```diff
@@ -118,9 +156,83 @@ <h1 id="title">Login</h1>
...
     font-family: Arial, Helvetica, sans-serif;
     font-size:1.15vw;
+    width: 175%;
   }
   
 
 </style>
+<script>
+  // List of sentences
+var _CONTENT = [
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:166]: it is VERY BAD PRACTICE to use `var`; always use `const` unless a variable needs to be mutable, then use `let`. BUT NEVER USE CONST!!!

File: incskill_site/templates/login.html [hunk 4]
```diff
@@ -118,9 +156,83 @@ <h1 id="title">Login</h1>
...
+  // List of sentences
+var _CONTENT = [ 
+	"Unlocking high-growth career", 
+	"Achieving opportunities", 
+	"Irrespective of past experience or needs", 
+	
+];
+
+// Current sentence being processed
+var _PART = 0;
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:174]: why are there `_` before all of these names you should remove that

File: incskill_site/templates/login.html [hunk 5]
```diff
@@ -118,9 +156,83 @@ <h1 id="title">Login</h1>
...
+
+// Character number of the current sentence being processed 
+var _PART_INDEX = 0;
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:177]: javascript variables names should be `camelCase` not `SCREAMING_SNAKE_CASE`

File: incskill_site/templates/login.html [hunk 6]
```diff
@@ -50,7 +58,35 @@ <h1 id="title">Login</h1>
...
+	
+	color: rgb(255, 255, 255);
+	letter-spacing: 2px;
+}
+
+#cursor {
+	display: inline-block;
+	
+	width: 3px;
+	height: 40px;
```
Comment (cubedhuang) [inline incskill_site/templates/login.html:78]: cursor should probably be relative to the current font size so use `em` for this unit

**Review by afry4321 — CHANGES_REQUESTED** [review]
The animation looks great! Please implement Daniel Huang's suggestions nd be sure to pull from origin because of new stuff in main. Let me check afterward.

File: incskill_site/templates/login.html (hunk 6 above)
Comment (stri4759) [reply incskill_site/templates/login.html:78]: hey daniel I cant use const, I have to use var

Comment (cubedhuang) [reply incskill_site/templates/login.html:78]: > hey daniel I cant use const, I have to use var

don't use `var`, use `let`; var has a lot of issues with function block scoping and hoisting

**Review by afry4321 — CHANGES_REQUESTED** [review]
Please pull from origin to get everything from main!

**Pull request merged**